[All Hosts] (manifest) update Runtimes element (#2628)
OfficeDev/office-js-docs-pr: docs/reference/manifest/runtimes.md

Specifies the runtime of your add-in. Child of the [`<Host>`](host.md) element.

> [!NOTE]

> > When running in Office on Windows, an add-in that has a `<Runtimes>` element in its manifest does not necessarily run in the same webview control as it otherwise would. The article [Browsers used by Office Add-ins](../../concepts/browsers-used-by-office-web-add-ins.md) describes how the versions of Windows and Office determine what webview control is normally used. If the conditions described there for using Microsoft Edge with WebView2 (Chromium-based) are met, then the add-in uses that browser whether or not it has a `<Runtimes>` element. However, when those conditions are not met, an add-in with a `<Runtimes>` element always uses Internet Explorer 11 regardless of the Windows or Microsoft 365 version.

Reviewer: ```suggestion
> When running in Office on Windows, an add-in that has a `<Runtimes>` element in its manifest does not necessarily run in the same webview control as it otherwise would. For more information about how the versions of Windows and Office determine what webview control is normally used, see [Browsers used by Office Add-ins](../../concepts/browsers-used-by-office-web-add-ins.md). If the conditions described there for using Microsoft Edge with WebView2 (Chromium-based) are met, then the add-in uses that browser whether or not it has a `<Runtimes>` element. However, when those conditions are not met, an add-in with a `<Runtimes>` element always uses Internet Explorer 11 regardless of the Windows or Microsoft 365 version.
```
For accessibility, describe the link content before the link. That's why we use the "For more information about blah, see [blah].
